templates/pages/page--taxonomy--term.html.twig

```diff
@@ -1,11 +1,17 @@
 <main role="main">
   <a id="main-content" tabindex="-1"></a>
-  <div class="layout-content p-12 pt-6 border-t">
-    <div class="">
-      {{ page.content.idcui_page_title }}
+  <div class="layout-content py-6 container mx-auto">
+
+    <div class="text-3xl">
+      {{ page['#title']}}
+      {# {% if page.content.idcui_page_title %}
```

Comment:
no prob if you want to leave to document how things were handled, but i'd otherwise say let's delete unused code